src/LSST/util/Log.h

```diff
@@ -141,6 +141,15 @@ class Log {
     /// BUFFER or MIRRORED as the destination.
     void setMaxBuffers(int maxBuffers);
 
+    /// @return value of environment variabe LOGLVL, defaults to TRACE (1).
+    /// Expected values of LOGLVL are integers.

```

Comment:
Please consider to explain a little bit of expected value here. Do you support the Level 1 only or 1 to 3 or higher? Thanks!

Reply:
I'll add a comment. It's constrained to legitimate values. Values < TRACE (1) set to TRACE. Values > CRITICAL set to CRITICAL.